src/yadism/coefficient_functions/__init__.py

```diff
@@ -69,7 +69,7 @@ def light_component(self):
         comp.extend(light.kernels.generate(self.esf, nf))
         # instead missing encodes mass effects and so we need to fork:
         for ihq in range(nf + 1, 7):
-            if masses[ihq]:
+            if not masses[ihq]:
```

Comment:
This I believe to be plainly wrong :)

However, since we are computing single observables, and they have a single mass (we always said we are not taking into account 2-masses effects, i.e. two different massive particles in the same diagram), we should stop iterating `ihq`. We can just specify the `ihq` of the massive particle, and the numerical value of its mass, as inputs in the runcard.

Before, the motivation for iterating was exactly the FONLL implementation. But now that FONLL is not any longer in `yadism`, we should stop doing it.

Reply:
Yes you're right, this changes the FFNS result which should not be changed.

For some reason the `heavy.kernels.generate_missing` and `asy.kernels.generate_missing_asy` don't agree in the large-Q limit so I'm still trying to understand why that is.

Reply:
> For some reason the `heavy.kernels.generate_missing` and `asy.kernels.generate_missing_asy` don't agree in the large-Q limit so I'm still trying to understand why that is.

I guessed it was for this reason, as the previous commits, but it was worth to point out that this couldn't work, even to avoid you wasting too much time on this option.